packages/next-css/css-loader-config.js

```diff
@@ -1,4 +1,5 @@
 const ExtractCssChunks = require('extract-css-chunks-webpack-plugin')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
```

Comment:
This can be required only in production as per https://spectrum.chat/thread/9f9f43b8-ec8b-45e5-a8e3-5b57a62e9e67?m=MTU0MTUyNjk1OTcxOQ==

Reply:
like this:

```js
if (!dev) {
  const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
}
```

I think that is not good way. See: https://eslint.org/docs/rules/global-require

Reply:
> If you have a module that must be initialized with information that comes from the file-system or if a module is only used in very rare situations and will cause significant overhead to load it may make sense to disable the rule. If you need to require() an optional dependency inside of a try/catch, you can disable this rule for just that dependency using the // eslint-disable-line global-require comment.


Specifically

>  if a module is only used in very rare situations and will cause significant overhead to load it may make sense to disable the rule.

It'll cause faster bootup on development.